dataView/DataView.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2018 Extremely Heavy Industries Inc.
+ */
+
+import {Component} from 'react';
+import {hoistComponent, elemFactory} from 'hoist/core';
+import {grid} from 'hoist/grid';
+import {GridModel} from 'hoist/grid';
+import {baseCol} from 'hoist/columns/Core';
+
+@hoistComponent()
+class DataView extends Component {
+
+    render() {
+        const {store, selection, contextMenuFn, itemFactory} = this.model,
+            {rowCls, rowHeight} = this.props;
+
+        return grid({
```

Comment:
Can we set this grid model up in the constructor?  We don't expect to handle changes for any of these props?
Props are passed to component constructor and recreating in render seems risky -- if we are ever forced to re-render this component, we throw out all of our state!